consensus/parlia/parlia.go

```diff
@@ -426,6 +580,14 @@ func (p *Parlia) verifyCascadingFields(chain consensus.ChainHeaderReader, header
 		return fmt.Errorf("invalid gas limit: have %d, want %d += %d", header.GasLimit, parent.GasLimit, limit)
 	}
 
+	// Verify vote attestation for fast finality.
```

Comment:
```
		if chain.Config().IsLynn(header.Number) {
			if err := p.verifyVoteAttestation(chain, header, parents); err != nil {
                         }
		}
```
How about this? `chain.Config().IsLynn` is cheaper.

Reply:
I would rather not, after upgrading, the chain.Config().IsLynn would do twice if we write like this, which is not cheaper, and I want to mask this code in the inner function to make the original code looks simple.